lib/open_food_network/order_cycle_management_report.rb

```diff
@@ -46,34 +46,21 @@ def header
     end
 
     def search
-      if FeatureToggle.enabled?(:customer_balance, @user)
-        Spree::Order.
-          finalized.
-          not_state(:canceled).
-          distributed_by_user(@user).
-          managed_by(@user).
-          search(params[:q])
-      else
-        Spree::Order.
-          complete.
-          where("spree_orders.state != ?", :canceled).
-          distributed_by_user(@user).
-          managed_by(@user).
-          search(params[:q])
-      end
+      Spree::Order.
+        finalized.
+        not_state(:canceled).
+        distributed_by_user(@user).
+        managed_by(@user).
+        search(params[:q])
```

Comment:
This is slightly different, right? It uses `finalized` now instead of `complete` as the scope?

Reply:
But with the new feature it used finalized as well. So this is correctly enabling the feature. We want `finalized` now to track cancelled orders as well.

Reply:
Spot on @mkllnk